doc/ota_updates/readme.rst

```diff
@@ -621,7 +624,13 @@ Stream Interface
 
 TODO describe Stream Interface
```

Comment:
Can probably drop the TODO since there is at least a modicum of description and example now.